test/future_execution_test.rb

```diff
@@ -94,13 +105,13 @@ module FutureExecutionTest
       end
 
       describe 'serializers' do
-        let(:save_and_load) do
```

Comment:
Ah, I first thouht this was a dead-code (missed the reference to it in the oirginal test), I've put it back